_posts/2021-01-26-converting-puppetized-dsc-modules.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: "Converting to use Puppetized DSC Modules"
+author: pmcmaw
+categories:
+  - dsc
+  - puppet
+  - modules
+tags:
+  - tooling
+  - dsc resources
+  - windows
+  - modules
+  - forge
+  - powershell gallery
+---
+
+## Why bother?
+
+The main reason we are advocating moving to this new and shiny method is that it provides you with a long list of advantages and new features. 
+We're literally just taking the underyling DSC resources, wrapping them and making them available to you.
+If there’s a problem calling the DSC resources from Puppet, we’ll work on it, but if there’s a problem with how those resources behave, that’s upstream.
+
+**Seriously though what is there to not love?**
+
+* Minimal module maintainence costs
+* Automated module building
+* Property by property reporting
+* VSCode Extension with intellisense
+* We have 68 modules on the Forge in the [DSC namespace][dsc-namespace]
+* Converting is simple
```

Comment:
Where possible, it's good practice to replace "simple" with another word or phrase, like "relatively straightforward"

- https://www.writethedocs.org/videos/prague/2018/don-t-say-simply-jim-fisher/
- https://www.cherryleaf.com/2018/01/dont-say-simply/

I'm _still_ trying to fix this in my own writing 😭

Reply:
They're great links, thanks for that. 
Today I learned "Not to use the term simple/simply when writing guides".